docs/products/kafka/get-started.md

```diff
@@ -4,19 +4,330 @@ sidebar_label: Get started
 keywords: [quick start]
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
 import CreateService from "@site/static/includes/create-service-console.md"
 
-Start using Aiven for Apache Kafka® by creating and configuring a service, connecting to it, and playing with sample data.
+Start using Aiven for Apache Kafka® by setting up and configuring a service, connecting to it, and managing your data.
+
+## Prerequisites
+
+- Access to the [Aiven Console](https://console.aiven.io)
+- [Aiven Provider for Terraform](https://registry.terraform.io/providers/aiven/aiven/latest/docs)
+- [Aiven CLI](/docs/tools/cli)
 
 ## Create an Aiven for Apache Kafka® service
 
-<CreateService serviceType="Apache Kafka®" />
+You can create your Kafka service using the Aiven Console, the Aiven CLI, or automate
+the process with Terraform.
+
+<Tabs groupId="group1">
+<TabItem value="1" label="Console" default>
+
+<CreateService serviceType="Apache Kafka®"/>
+
+</TabItem>
+<TabItem value="2" label="Terraform">
+
+1. [Create a personal token](/docs/platform/howto/create_authentication_token).
+1. Create a `sample.tf` file for the `aiven` provider configuration and the `aiven_kafka`
+   resource.
+
+   ```hcl
+   variable "aiven_token" {
+     type = string
+   }
+
+   variable "aiven_project_name" {
+     type = string
+   }
+
+   terraform {
+     required_providers {
+       aiven = {
+         source  = "aiven/aiven"
+         version = ">=4.0.0, <5.0.0"
+       }
+     }
+   }
+
+   provider "aiven" {
+     api_token = var.aiven_token
+   }
+
+   resource "aiven_kafka" "kafka" {
+     project                 = var.aiven_project_name
+     cloud_name              = "google-europe-west1"
+     plan                    = "startup-16"
+     service_name            = "my-kafka"
+     maintenance_window_dow  = "friday"
+     maintenance_window_time = "23:00:00"
+
+     kafka_user_config {
+       kafka_version = "2.7"
```

Comment:
I don't think kafka_version 2.7 is still supported so we might want to update that I'll find another terraform file I know was executed recently against the dev sandbox

Reply:
Actually it wasnt even specified here so I guess it used the default
https://github.com/aiven/governance-github-discovery/blob/main/kafka-resources.tf

But looking at dev-sandbox version 3.7 is default and 3.9 is the latest maybe we should pick the default for the moment?